docs/wiki/framework/missile-guidance-framework.md

```diff
@@ -109,6 +129,17 @@ class ace_missileguidance_attackProfiles {
         description = "";  // Description
 
         functionName = "my_fnc_doAttackProfile";  // Function that handles the attack profile
+        onFired = "my_fnc_onFired"; // Function that runs when missile is fired using this attack profile
+    };
+};
+```
+
+### 4.4 Custom Navigation Type
+```cpp
```

Comment:
```suggestion
```sqf
```
Doesn't the SQF highlighting work here, too ?

Reply:
No. And besides, this is config, not SQF.

Reply:
Good to know.